src/test/resources/hudson/tasks/junit/JENKINS-41134/TestSuite_second.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites>
+    <testsuite name="TestSuite_second" tests="1" errors="0" failures="1" skip="0" timestamp="2012-04-05T10:50:00">

```

Comment:
True, I will add another test. I did verify on an actual build of Jenkins with the Jenkinsfile that the fix does resolve that issue, but a completely valid test is a good idea.